temporal/t.rast.aggregate/t.rast.aggregate.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python3
-
 ############################################################################
```

Comment:
```suggestion

############################################################################
```

(re-add missing blank line)